Dockerfile

```diff
@@ -204,10 +204,9 @@ ARG LIBMYSOFA_URL="https://github.com/hoene/libmysofa/archive/refs/tags/v$LIBMYS
 ARG LIBMYSOFA_SHA256=94cb02e488de4dc0860c8d23b29d93d290bb0a004d4aa17e1642985bba158ee9
 # bump: uavs3d /UAVS3D_COMMIT=([[:xdigit:]]+)/ gitrefs:https://github.com/uavs3/uavs3d.git|re:#^refs/heads/master$#|@commit
 # bump: uavs3d after ./hashupdate Dockerfile UAVS3D $LATEST
-# bump: uavs3d after COMMIT=$(git ls-remote https://github.com/uavs3/uavs3d.git $LATEST^{} | awk '{print $1}') && sed -i -E "s/^ARG UAVS3D_COMMIT=.*/ARG UAVS3D_COMMIT=$COMMIT/" Dockerfile
```

Comment:
@mathieu-aubin the reason i do this for aom bump is because i want to checkout a specific version (i think it does not build with proper version number etc if not) but still wanted to verify the commit hash. Wish i had time to make bump a bit more powerful to make things like this easier.